### PR TITLE
fix(aws): rework CD docs

### LINKED ISSUE #129: AWS Continuous deployment documentation is misleading
- forking does not set up webhook listeners
- changing repository URLs does not set up AWS CodePipeline
- *Provide GitHub credentials* and *Enable continuous deployment* is part of the necessary steps
https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/saga/docs/aws/ContinuousDeployment.html

### PR DESCRIPTION
**Pull request by coderbyheart**
Fixes #129

### CODE REVIEW

File: docs/aws/ContinuousDeployment.rst [hunk 1]
```diff
@@ -7,46 +7,43 @@ Continuous deployment
...
 
-A second CodePipeline will be set up for the web application, which triggers a CodeBuild project for every push to the :file:`saga` branch.
-Configure the repository URL and the branch for the web application in the ``deploy.webApp`` property of the :file:`package.json` file.
-The CodeBuild project upgrades the web application deployment on the S3 bucket.
+#. Update the `deploy.webApp.repository <https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js/blob/3ddf0c117de7f22f02df46c3e1ca899915627d46/package.json#L144>`_ in the :file:`package.json` file of your nRF Asset Tracker for AWS fork. It should point to the repository URL of your fork of the Cat Tracker web application.
 
 Provide GitHub credentials
 **************************
 
+This is a manual one-time step that is needed so the AWS CodePipeline project created when enabling continuous deployment below is able to register a webhook in the GitHub repository of your fork to be notified about changes.
```
Comment (peknis) [inline docs/aws/ContinuousDeployment.rst]: This is a very long and complex sentence. How about something like this:

When you enable the continuous deployment, you create an AWS CodePipeline project.
This is a manual one-time step to enable the AWS CodePipeline project to register a webhook in the GH repository of your fork to be notified about changes.

File: docs/aws/ContinuousDeployment.rst [hunk 2]
```diff
@@ -57,6 +54,17 @@ After providing the GitHub credentials, set up the continuous deployment by enab
     node cli configure-api context stack cd 1
     npx cdk deploy '*'
 
+This sets up an AWS CodePipeline, which triggers an AWS CodeBuild project for every push to the ``saga`` branch.
+You can customize the branch by providing the name in ``deploy.branch`` in the :file:`package.json` file of your nRF Asset Tracker for AWS fork.
+
+A second AWS CodePipeline will be set up for the web application, which triggers a CodeBuild project for every push to the :file:`saga` branch.
+Configure the repository URL and the branch for the web application in the ``deploy.webApp`` property of the :file:`package.json` file of your nRF Asset Tracker for AWS fork.
```
Comment (peknis) [inline docs/aws/ContinuousDeployment.rst]:
```suggestion
Another AWS CodePipeline is set up for the web application.
It triggers a CodeBuild project for every push to the :file:`saga` branch.
Configure the repository URL and the branch for the web application in the ``deploy.webApp`` property of the :file:`package.json` file of your nRF Asset Tracker for AWS fork.
```

File: docs/aws/ContinuousDeployment.rst [hunk 3]
```diff
@@ -7,46 +7,43 @@ Continuous deployment
    :local:
    :depth: 2
 
-You can enable continuous deployment of the changes in your source repository to an AWS account.
+You can deploy all changes that you make to a fork of the nRF Asset Tracker for AWS automatically.
```
Comment (peknis) [inline docs/aws/ContinuousDeployment.rst]:
```suggestion
You can automatically deploy all changes that you make to a fork of the nRF Asset Tracker for AWS.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

File: docs/aws/ContinuousDeployment.rst [hunk 4]
```diff
@@ -7,46 +7,45 @@ Continuous deployment
...
 
-For the continuous deployment to work, complete the following steps:
+Fork the nRF Asset Tracker repositories
+***************************************
 
-1. Fork the source code to register the webhook listener that triggers an upgrade of your deployment.
+To enable continuous deployment, complete the following steps:
 
-#. Make sure to update the ``repository.url`` in the :file:`package.json` file in your fork.
+1. Fork `the nRF Asset Tracker for AWS repository <https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js>`_.
```
Comment (umapraseeda) [inline docs/aws/ContinuousDeployment.rst]:
```suggestion
1. Fork the `nRF Asset Tracker for AWS repository <https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js>`_.
```

File: docs/aws/ContinuousDeployment.rst [hunk 5]
```diff
@@ -7,46 +7,45 @@ Continuous deployment
...
 
-   This sets up an AWS CodePipeline, which triggers a CodeBuild project for every push to the ``saga`` branch.
-   
-#. Configure the branch in the ``deploy.branch`` property of the :file:`package.json` file.
+#. Update the `repository.url <https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js/blob/3ddf0c117de7f22f02df46c3e1ca899915627d46/package.json#L15>`_ in the :file:`package.json` file in your fork. It should point to the repository URL of your fork.
```
Comment (umapraseeda) [inline docs/aws/ContinuousDeployment.rst]:
```suggestion
#. Update the `repository.url <https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js/blob/3ddf0c117de7f22f02df46c3e1ca899915627d46/package.json#L15>`_ in the :file:`package.json` file in your fork. It must point to the repository URL of your fork.
```

File: docs/aws/ContinuousDeployment.rst [hunk 6]
```diff
@@ -7,46 +7,45 @@ Continuous deployment
...
 
-   The CodeBuild project upgrades the CloudFormation stack, which contains the nRF Asset Tracker resources.
+#. Fork `the Cat Tracker web application repository <https://github.com/NordicSemiconductor/asset-tracker-cloud-app-js>`_.
```
Comment (umapraseeda) [inline docs/aws/ContinuousDeployment.rst]:
```suggestion
#. Fork the `Cat Tracker web application repository <https://github.com/NordicSemiconductor/asset-tracker-cloud-app-js>`_.
```

File: docs/aws/ContinuousDeployment.rst [hunk 7]
```diff
@@ -7,46 +7,45 @@ Continuous deployment
...
 
-A second CodePipeline will be set up for the web application, which triggers a CodeBuild project for every push to the :file:`saga` branch.
-Configure the repository URL and the branch for the web application in the ``deploy.webApp`` property of the :file:`package.json` file.
-The CodeBuild project upgrades the web application deployment on the S3 bucket.
+#. Update the `deploy.webApp.repository <https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js/blob/3ddf0c117de7f22f02df46c3e1ca899915627d46/package.json#L144>`_ in the :file:`package.json` file of your nRF Asset Tracker for AWS fork. It should point to the repository URL of your fork of the Cat Tracker web application.
```
Comment (umapraseeda) [inline docs/aws/ContinuousDeployment.rst]:
```suggestion
#. Update the `deploy.webApp.repository <https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js/blob/3ddf0c117de7f22f02df46c3e1ca899915627d46/package.json#L144>`_ in the :file:`package.json` file of your nRF Asset Tracker for AWS fork. It must point to the repository URL of your fork of the Cat Tracker web application.
```

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**